ts-graphql-yoga/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "declaration": true,
+      "removeComments": false,
+      "emitDecoratorMetadata": true,
+      "experimentalDecorators": true,
+      "allowSyntheticDefaultImports": true,
+      "target": "ES2019",
+      "sourceMap": true,
+      "outDir": "./dist",
+      "baseUrl": "./",
+      "incremental": true,
+      "skipLibCheck": true,
+      "strictNullChecks": false,
+      "noImplicitAny": false,
+      "strictBindCallApply": false,
+      "forceConsistentCasingInFileNames": false,
+      "noFallthroughCasesInSwitch": false,
+      "moduleResolution": "node"
+    },
+    "exclude": ["node_modules", "test", "dist", "**/*spec.ts", "compiled"]
```

```suggestion
    "exclude": ["node_modules", "test", "dist", "**/*spec.ts", "compiled*"]
```

suggestion, in case they want to play with different configs, each to a different dir

---

Is that just assuming that they'll want to prefix each output dir with `compiled`? I don't know if I'd make that assumption. 🤷‍♂️

---

the repo's gitignore does, fwiw: https://github.com/KlothoPlatform/sample-apps/blob/main/.gitignore#L3

---

I guess I'm good with that.

---

I often use `compiled_old` and `compiled_new` to diff between compilations. The base gitignore is to accommodate that to make sure I don't check in compiled code 😄